geps/gep-713.md

```diff
@@ -910,7 +910,115 @@ plugin.
 This is already part of the API pattern, but is being lifted to more prominience
 here.
 
-#### Standard status Condition
+#### Standard status struct
+
+Status: Recommended
+
+Policy objects SHOULD use the upstream `PolicyAncestorStatus` struct in their respective
+Status structs. Please see the included `PolicyAncestorStatus` struct, and its use in
+the `BackendTLSPolicy` object for detailed examples. Included here is a representative
+version.
+
+This is an `Ancestor` status rather than a `Parent` status, as in the Route status
+because for Policy attachment, the relevant object may or may not be the direct
+parent.
+
+For example, `BackendTLSPolicy` directly attaches to a Service, which may be included
+in multiple Routes, in multiple Gateways. However, the status of the `BackendTLSPolicy`
+will be different only at the Gateway level, so Gateway is the relevant Ancestor
+for the status.
+
+Each Gateway that has a Route that includes a backend with an attached `BackendTLSPolicy`
+MUST have a separate `PolicyAncestorStatus` section in the `BackendTLSPolicy`'s
+`status.ancestors` stanza, which mandates that entries must be distinct using the
+combination of the `AncestorRef` and the `ControllerName` fields as a key.
+
+See [GEP-1897][gep-1897] for the exact details.
+
+[gep-1897]: /geps/gep-1897
+
+```go
+// PolicyAncestorStatus describes the status of a route with respect to an
+// associated Ancestor.
+//
+// The object that a Policy attaches to (the object in its TargetRef) is its
+// _parent_, and any other objects above it in an object hierarchy are its
+// _ancestors_.
```

Comment:
Maybe something like this?
```suggestion
// Ancestors refer to objects that are either the Target of a policy or above it in terms
// of object hierarchy. For example, if a policy targets a Service, an Ancestor could be
// a Route or a Gateway. 

// In the context of policy attachment, the Ancestor is used to distinguish which
// resource results in a distinct application of this policy. For example, if a policy
// targets a Service, it may have a distinct result per attached Gateway.
```